### PR TITLE
Add Final Report for "BMv2 PNA Support"

### PR DESCRIPTION
**Pull request by rupesh-chiluka-marvell**

### CODE REVIEW

**Review by Dscano — COMMENTED** [review]
I have also 2 proposals:

- Is it possible add an image that shows the PNA architetcure. More over,  only if it is feasible,  it  would be nice highlight in the image what you have implemented of PNA.
- Can you add a "video/gif" that shows statup of BMv2 PNA? Furthermore, the "video/gif" should display the BMv2 PNA that you have to run and show that the BMv2 PNA reach the "steady state".

File: 2024/projects/bmv2_pna/README.md [hunk 1]
```diff
@@ -1 +1,127 @@
...
+  - [Future Work](#future-work)
+    - [Standard PNA Features (Implemented)](#standard-pna-features-implemented)
+    - [Standard PNA Features (Yet to be Implemented)](#standard-pna-features-yet-to-be-implemented)
+    - [Experimental PNA Features (Yet to be Implemented)](#experimental-pna-features-yet-to-be-implemented)
+
+## Abstract
+
+As the P4 use cases on the NIC side increase, so does the need for a P4 simulator that supports [Portable NIC Architecture (PNA)](https://p4.org/p4-spec/docs/PNA-v0.7.html). Currently, the BMv2 simulator only supports the v1model and [Portable Switch Architecture (PSA)](https://p4.org/p4-spec/docs/PSA-v1.2.html) specifications. This project aims to integrate a minimal core subset of the PNA into the BMv2 and the P4C. The PNA Architecture Block Diagram is shown in below:
+
+<picture>
```
Comment (Dscano) [inline 2024/projects/bmv2_pna/README.md:29]: Could you center the image?

Comment (rupesh-chiluka-marvell) [reply 2024/projects/bmv2_pna/README.md]: Ok

**Review by Dscano — APPROVED** [review]
LGTM

File: 2024/projects/bmv2_pna/README.md [hunk 2]
```diff
@@ -1 +1,127 @@
...
+
+- The [pna-demo-L2-one-table.p4](https://github.com/p4lang/pna/blob/main/examples/pna-demo-L2-one-table.p4) program from `p4lang/pna` repository is used to test the new PNA NIC target.
+
+- Compile `pna-demo-L2-one-table.p4` using `p4c-bm2-pna` binary.
+  - `p4c-bm2-pna pna-demo-L2-one-table.p4 -o pna-demo-L2-one-table.json`
+
+- To start the `pna_nic` target:
+
+<p align="center">
+  <img alt="PNA Block Diagram" src="assets/pna_nic.gif" width="500">
```
Comment (qobilidop) [inline 2024/projects/bmv2_pna/README.md:83]: Nit: Could change the `alt` text?

**Review by qobilidop — APPROVED** [review]
LGTM now. Thanks @rupesh-chiluka-marvell!

File: 2024/projects/bmv2_pna/README.md [hunk 3]
```diff
@@ -1 +1,127 @@
...
+- Meters
+- Hash
+- Internet Checksum
+- Random
+
+### Standard PNA Features (Yet to be Implemented)
+- Recirculation
+- Dropping
+- Differentiate between Host and Network Interfaces
+- Drop
```
Comment (qobilidop) [inline 2024/projects/bmv2_pna/README.md:121]: Any difference between Dropping and Drop above?

Comment (rupesh-chiluka-marvell) [reply 2024/projects/bmv2_pna/README.md:121]: Sorry,
Both are the same.
Will remove it

File: 2024/projects/bmv2_pna/README.md (hunk 2 above)
Comment (rupesh-chiluka-marvell) [reply 2024/projects/bmv2_pna/README.md:83]: Yes
Will change to pna_cli

**Pull request merged**